IdentityCore/src/cache/MSIDToken.m

```diff
@@ -368,8 +368,8 @@ - (void)fillAdditionalServerInfoFromResponse:(MSIDTokenResponse *)tokenResponse
     if ([tokenResponse isKindOfClass:[MSIDAADTokenResponse class]])
     {
         MSIDAADTokenResponse *aadTokenResponse = (MSIDAADTokenResponse *)tokenResponse;
-        [serverInfo setObject:aadTokenResponse.extendedExpiresIn forKey:@"ext_expires_on"];
```

Comment:
will there be a case in which these fields are nil for the key?

Reply:
yes, that's why I changed it to setValue